[2022-06-02 12:18:46,307] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: parallel_runs.task_4 manual__2022-06-02T06:48:27.422213+00:00 [queued]>
[2022-06-02 12:18:46,313] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: parallel_runs.task_4 manual__2022-06-02T06:48:27.422213+00:00 [queued]>
[2022-06-02 12:18:46,313] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-02 12:18:46,313] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-02 12:18:46,314] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-02 12:18:46,323] {taskinstance.py:1378} INFO - Executing <Task(BashOperator): task_4> on 2022-06-02 06:48:27.422213+00:00
[2022-06-02 12:18:46,343] {standard_task_runner.py:52} INFO - Started process 15883 to run task
[2022-06-02 12:18:46,348] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_runs', 'task_4', 'manual__2022-06-02T06:48:27.422213+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/parallel_runs.py', '--cfg-path', '/tmp/tmpdw8faa8x', '--error-file', '/tmp/tmpgl_hjqwu']
[2022-06-02 12:18:46,349] {standard_task_runner.py:80} INFO - Job 32: Subtask task_4
[2022-06-02 12:18:46,383] {task_command.py:370} INFO - Running <TaskInstance: parallel_runs.task_4 manual__2022-06-02T06:48:27.422213+00:00 [running]> on host TIGER0990.tigeranalytics.local
[2022-06-02 12:18:46,420] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_runs
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2022-06-02T06:48:27.422213+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-02T06:48:27.422213+00:00
[2022-06-02 12:18:46,421] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-02 12:18:46,421] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-06-02 12:18:46,447] {subprocess.py:85} INFO - Output:
[2022-06-02 12:18:49,459] {subprocess.py:96} INFO - Command exited with return code 0
[2022-06-02 12:18:49,520] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=parallel_runs, task_id=task_4, execution_date=20220602T064827, start_date=20220602T064846, end_date=20220602T064849
[2022-06-02 12:18:49,566] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-02 12:18:49,601] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
